sdk/communication/azure-communication-common/build.gradle

```diff
@@ -1,6 +1,6 @@
 ext.publishName = "Microsoft Azure Android Client Library common code For Communication Service"
 description = "This package contains the Android client library common code for Microsoft Azure Communication Service."
-version = "1.0.0-beta.1"
+version = "1.0.0-beta.3"
```

Comment:
Should this say  `1.0.0-beta.2`?

Reply:
beta.2 was the version we just published for communication in general. We want the current to be beta.3 (Unreleased). But Common is a special case where we didn't release beta.2. So the version has to jump to be consistent on next release.